src/Validator/validator.js

```diff
@@ -0,0 +1,80 @@
+import { ERROR, LOTTO } from '../Constants/constants.js';
+import CustomError from '../Error/CustomError.js';
+
+class validator {
+  static validatePurchaseAmount(amount) {
+    this.validatePurchaseAmountDivided(amount);
+    this.validatePurchaseAmountIsNumber(amount);
+  }
+
+  static validatePurchaseAmountDivided(amount) {
+    if ((amount * 1000) % 1000 !== 0) {
+      throw new CustomError(ERROR.PURCHASE_AMOUNT.NOT_PRIME_NUMBER);
+    }
+  }
+
+  static validatePurchaseAmountIsNumber(amount) {
+    if (isNaN(amount)) {
+      throw new CustomError(ERROR.PURCHASE_AMOUNT.IS_NOT_NUMBER);
+    }
+  }
+
+  static validateLottoNumbers(numbers) {
+    this.validateLottoNumbersLength(numbers);
+    this.validateLottoNumbersDuplicate(numbers);
+    this.validateLottoNumbersIsNumber(numbers);
+    this.validateLottoNumbersRange(numbers);
+  }
+
+  static validateLottoNumbersLength(numbers) {
+    if (numbers.length !== 6) {
+      throw new CustomError(ERROR.LOTTO_NUMBERS.INVALID_LENGTH);
+    }
+  }
+
+  static validateLottoNumbersDuplicate(numbers) {
+    if (new Set(numbers).size !== LOTTO.LOTTO_NUMBERS) {
+      throw new CustomError(ERROR.LOTTO_NUMBERS.IS_DUPLICATE);
+    }
+  }
+
+  static validateLottoNumbersIsNumber(numbers) {
+    numbers.forEach(number => {
+      if (isNaN(number)) {
```

Comment:
코드 컨벤션에 따라 Number.isNaN을 사용해보시는 것은 어떨까요? 🙌
```js
[29.1](https://github.com/airbnb/javascript#standard-library--isnan) 

Use Number.isNaN instead of global isNaN. eslint: [no-restricted-globals](https://eslint.org/docs/rules/no-restricted-globals)

Why? The global isNaN coerces non-numbers to numbers, returning true for anything that coerces to NaN. If this behavior is desired, make it explicit.

// bad
isNaN('1.2'); // false
isNaN('1.2.3'); // true

// good
Number.isNaN('1.2.3'); // false
Number.isNaN(Number('1.2.3')); // true
```